api/indexer.oas2.json

```diff
@@ -1613,16 +1711,38 @@
         }
       }
     },
-    "ParticipationUpdates": {
```

Comment:
The `ParticipationUpdates` has been moved further below to starting line 1834 and following the `OnCompletion` section